src/SmartComponents/ProviderPage/providerForm.js

```diff
@@ -37,9 +37,9 @@ export const wizardForm = {
                 name: 'step-1',
                 stepKey: 1,
                 nextStep: {
-                    when: 'source-type',
+                    when: 'source_type',
```

Comment:
What is this case handling that can't be gotten from the source_type records?

Reply:
This will be gotten from the source_type records in the future.

Reply:
Okay can we do this as a follow up now?  I don't think there was any urgency to get this in immediately but could be wrong